config/locales/en.yml

```diff
@@ -1106,9 +1086,6 @@ en:
     enroll_confirmation: Are you sure you want to add %{username}?
     enroll_url: 'Users may join by visiting this URL:'
     enrolled_success: '%{username} was added successfully!'
-    exercise_modules_completed:
```

Comment:
Looks like you've deleted a lot of interface messages on accident.

Reply:
I guess this may be related to starting from an older version of the code, and then overwriting the changes the happened in the meantime with your version of en.yml. I recommend starting from the latest version and re-adding your new i18n messages.

Reply:
This time I made my changes over latest `en.yml` file.